src/sas/qtgui/Calculators/media/shape_2_sas_help.rst

```diff
@@ -0,0 +1,87 @@
+.. shape_2_sas_help.rst
+
+.. by J Krzywon, NIST, May 2025
+
```

Comment:
Typically this is placed at the very end as:

```rst
.. note::
    This help document was last changed by Steve King, 08Oct2016
```
The idea is to be completely transparent as to the age of the documentation as well as the author in case questions arise.